includes/admin/tasks/class-wc-payments-task-disputes.php

```diff
@@ -275,6 +284,10 @@ public function is_complete() {
 	 * @return bool
 	 */
 	public function can_view() {
+		if ( null === $this->disputes_needing_response ) {
+			// init() is called on can_view as this is the first method called in the task lifecycle by WC.
```

Comment:
If we rename `init` based on what it does – i.e. fetch stuff so we have the data we need – then this comment can be clearer.